repos/fdbt-site/cypress_tests/cypress/e2e/capped.cy.ts

```diff
@@ -1,44 +1,41 @@
-import { isFinished } from '../support/helpers';
+import { getRandomNumber, isFinished } from '../support/helpers';
 import {
     defineUserTypeAndTimeRestrictions,
     selectFareType,
     completeCappedGeoZonePages,
     completeCappedDistanceJourney,
     completeCappedMultiServicePages,
     completeCappedTapsJourney,
+    completeCappedProductJourney,
 } from '../support/steps';
 
 
 describe('the capped faretype product journey', () => {
-    it('completes successfully for geozone distance journey', () => {
-        selectFareType('cappedProduct', false);
-        defineUserTypeAndTimeRestrictions();
-        completeCappedGeoZonePages()
-        completeCappedDistanceJourney()
-        isFinished()
-    });
 
-    it('completes successfully for multi-service distance journey', () => {
+    const isGeoZoneJourney = getRandomNumber(1, 2) === 1 ? true: false;
```

Comment:
We need this on a 'per test' basis, because if it decides geoZone then it will do them *all* as geoZone. We want it to randomly decide in each test. Dont worry about the name of the test, the name of the test should just be which *capped* journey its taking.